integrated_channels/sap_success_factors/transmitters/__init__.py

```diff
@@ -0,0 +1,13 @@
+"""
+Package for transmitting data to SuccessFactors.
+"""
+from __future__ import unicode_literals
+
+
+class BaseSuccessFactorsTransmitter(object):
```

Comment:
It depends on the language, but typically I would expect to see "Base" at the end of the name.